executor/cluster_reader_test.go

```diff
@@ -55,6 +55,15 @@ func (s *testClusterReaderSuite) TearDownSuite(c *C) {
 	s.store.Close()
 }
 
+func (s *testClusterReaderSuite) TestMetricTableData(c *C) {
+	c.Assert(failpoint.Enable("github.com/pingcap/tidb/executor/mockMetricRetrieverQueryPromQL", "return"), IsNil)
+	defer func() {
+		c.Assert(failpoint.Disable("github.com/pingcap/tidb/executor/mockMetricRetrieverQueryPromQL"), IsNil)
+	}()
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustQuery("select * from metric_schema.query_duration;").Check(testkit.Rows("2019-12-23 20:11:35.000000 0.1 127.0.0.1:10080  0.9"))
```

Comment:
Please put the result into new line to make review easily.

Reply:
I think we should more test cases to cover multiple quantiles.

Reply:
done.